MUXSDKImaListener/Classes/MuxImaListener.m

```diff
@@ -114,6 +115,9 @@ - (void)onContentPauseOrResume:(bool)isPause {
         [self setupAdViewData:playbackEvent withAd:nil];
         [_playerBinding dispatchAdEvent: playbackEvent];
         
+        _sendAdplayOnStarted = NO;
+        [_playerBinding dispatchAdEvent: [MUXSDKAdPlayEvent new]];
```

Comment:
`[[MUXSDKAdPlayEvent alloc] init]` should be used here instead of `new`.

https://macresearch.org/difference-between-alloc-init-and-new/

Reply:
I updated the one I added, but most of the class uses `new`. Would you want to fix/validate the rest of them during the refactor to move the rest of the private fields proposed [here](https://github.com/muxinc/mux-stats-google-ima/pull/32#discussion_r1546735239)?

Reply:
Thank you! Yes, that sounds like a good plan.